packages/services/api/src/modules/auth/lib/authz.spec.ts

```diff
@@ -213,4 +213,32 @@ describe('Session.assertPerformAction', () => {
       .catch(err => err);
     expect(result2).toBeInstanceOf(AccessError);
   });
+
+  test('Allow if no resource was specified on the permission', async () => {
+    const orgId = 'aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa';
+    const bId = 'bbbbbbbb-bbbb-bbbb-bbbb-bbbbbbbbbbbb';
+    const cId = 'cccccccc-cccc-cccc-cccc-cccccccccccc';
+
+    const session = new TestSession([
+      {
+        effect: 'allow',
+        action: 'appDeployment:create',
+        resource: [],
```

Comment:
Supporting this allow an to create app deployments within any organization, this also breaks an existing integration test for permissions see [`prevent access to projects with assigned resources on member` ](https://github.com/graphql-hive/console/actions/runs/17136185798/job/48613650721?pr=6918)